graphql.go

```diff
@@ -132,7 +132,7 @@ func Tracer(tracer trace.Tracer) SchemaOpt {
 
 // ValidationTracer is used to trace validation errors. It defaults to trace.NoopValidationTracer.
 // Deprecated: context is needed to support tracing correctly. Use a Tracer which implements trace.ValidationTracerContext.
-func ValidationTracer(tracer trace.ValidationTracer) SchemaOpt {
+func ValidationTracer(tracer trace.ValidationTracer) SchemaOpt { //nolint:staticcheck
```

Comment:
Why ignoring `statickcheck`? What rule does this func violate?

Reply:
I mentioned this in the PR message:

```
graphql.go:135:30: SA1019: trace.ValidationTracer is deprecated: use ValidationTracerContext. (staticcheck)
func ValidationTracer(tracer trace.ValidationTracer) SchemaOpt {
                             ^
graphql.go:300:9: SA1019: trace.ValidationTracer is deprecated: use ValidationTracerContext. (staticcheck)
        tracer trace.ValidationTracer
```

This function is deprecated, so we need to add the exception.